pkg/config/policies.go

```diff
@@ -43,8 +43,8 @@ func (c *ConfigTx) UpdateConsortiumChannelCreationPolicy(consortiumName string,
 	return nil
 }
 
-// GetPoliciesForConsortiums returns a map of policies for channel consortiums.
-func (c *ConfigTx) GetPoliciesForConsortiums() (map[string]Policy, error) {
+// ConsortiumsPolicies returns a map of policies for channel consortiums.
+func (c *ConfigTx) ConsortiumsPolicies() (map[string]Policy, error) {
```

Comment:
This can be removed now that FAB-17665 is merged

Reply:
🛠